src/niweb/apps/noclook/models.py

```diff
@@ -130,6 +130,57 @@ def delete(self, **kwargs):
     delete.alters_data = True
 
 
+DEFAULT_ROLEGROUP_NAME = 'default'
+DEFAULT_ROLE_KEY = 'employee'
+DEFAULT_ROLES = {
+    'abuse_contact': { 'name': 'Abuse', 'description': '' },
```

Comment:
Doesn't other default data use fixtures?

Reply:
For this, instead of fixtures I used RunPython to init the default roles and rolegroup as it's recommended here: https://docs.djangoproject.com/en/1.11/howto/initial-data/#providing-initial-data-with-migrations . I also squashed all the migrations related with roles and rolegroup so the initial data starts with the right models.

Reply:
I haven't seen RunPython (or RunSQL) before. Nice to know!